flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -25,17 +25,33 @@ class SentryFlutterOptions extends SentryOptions {
   /// feature, as this is required to mark Sessions as Crashed.
   bool enableNativeCrashHandling = true;
 
-  int _autoSessionTrackingIntervalMillis = 30000;
-
   /// The session tracking interval in millis. This is the interval to end a
   /// session if the App goes to the background.
   /// See: [enableAutoSessionTracking]
+  @Deprecated('Use autoSessionTrackingInterval instead')
   int get autoSessionTrackingIntervalMillis =>
-      _autoSessionTrackingIntervalMillis;
+      autoSessionTrackingInterval.inMilliseconds;
 
+  @Deprecated('Use autoSessionTrackingInterval instead')
   set autoSessionTrackingIntervalMillis(int value) {
-    _autoSessionTrackingIntervalMillis =
-        value >= 0 ? value : _autoSessionTrackingIntervalMillis;
+    autoSessionTrackingInterval = value >= 0
+        ? Duration(milliseconds: value)
+        : autoSessionTrackingInterval;
+  }
+
+  Duration _autoSessionTrackingInterval = Duration(milliseconds: 30000);
+
+  /// The session tracking interval. This is the interval to end a
+  /// session if the App goes to the background.
+  /// Default is 30 seconds/30000 milliseconds.
+  /// See: [enableAutoSessionTracking]
+  /// Always uses the given duration as a positiv timespan.
+  Duration get autoSessionTrackingInterval => _autoSessionTrackingInterval;
+
+  set autoSessionTrackingInterval(Duration value) {
+    assert(!value.isNegative);
+    assert(value > Duration.zero);
```

Comment:
```suggestion
    assert(value > Duration.zero);
```

both asserts do the same thing right

Reply:
yep, you are right